CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+# Copyright 2021-2021 Leon Brands. All rights served.
+# License: https://github.com/Legion-Engine/Legion-LLRI/blob/main/LICENSE
+
+cmake_minimum_required(VERSION 3.13 FATAL_ERROR)
+
+# Folder structure
+set_property(GLOBAL PROPERTY USE_FOLDERS ON)
+set(CMAKE_SKIP_INSTALL_ALL_DEPENDENCY ON)
+set(CMAKE_FOLDER CMakePredefinedTargets)
+
+# Project
+project(llri LANGUAGES CXX)
+set(CMAKE_CONFIGURATION_TYPES Debug Release)
+
+# Easy folder references for subdirs
+set(LLRI_DIR_OUTPUT_INCLUDE ${CMAKE_CURRENT_SOURCE_DIR}/include)
+set(LLRI_DIR_OUTPUT_LIB ${CMAKE_CURRENT_SOURCE_DIR}/lib)
```

Comment:
these could be marked as `CACHE INTERNAL ""` or `PARENT_SCOPE` so that other projects can use them when importing the CMakeLists.txt

Reply:
Resolved